CanHazFunny/CanHazFunny.Tests/CanHazFunny.Tests.csproj

```
@@ -1,24 +1,33 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
+		<LangVersion>12.0</LangVersion>
+		<Nullable>enable</Nullable>
+		<EnableNETAnalyzers>True</EnableNETAnalyzers>
+		<Nullability>True</Nullability>
+		<CodeAnalysisTreatWarningAsErrors>True</CodeAnalysisTreatWarningAsErrors>
+		<EnforceCodeStyleInBuild>True</EnforceCodeStyleInBuild>
+		<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
```

```suggestion
		<EnableNETAnalyzers>True</EnableNETAnalyzers>
		<Nullability>True</Nullability>
		<CodeAnalysisTreatWarningAsErrors>True</CodeAnalysisTreatWarningAsErrors>
		<EnforceCodeStyleInBuild>True</EnforceCodeStyleInBuild>
		<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
```
Set all 'True' to 'true'

---

thanks :3  @MannyPoptart